src/MedTrackerAPI.Tests/Features/Supplies/AddSupplyTests.cs

```diff
@@ -0,0 +1,61 @@
+using FluentValidation.TestHelper;
+using MedTrackerAPI.Features.Supplies;
+using MedTrackerAPI.Infrastructure;
+using static MedTrackerAPI.Tests.FakeFactory;
+
+namespace MedTrackerAPI.Tests.Features.Supplies;
+
+public class AddSupplyTests
+{
+    private Device _device;
+
+    [SetUp]
+    public void Setup()
+    {
+        _device = CreateFakeDevice();
+    }
+    
+    [TestCase(null)]
+    [TestCase("")]
+    public void GivenNoDescription_ThenIsNotValid(string? value) =>
+        new AddSupply.AddSupplyCommandValidator().TestValidate(new AddSupply.AddSupplyCommand
```

Comment:
I would add the static reference for AddSupply so you don’t need to add that each time.